### ISSUE TITLE
Status 16389

### ISSUE THREAD

**Issue by avery021** [jwijffels/taskscheduleR#23]
Hello,

I tried to run a daily R script using the rstudio add-in and receive the following error:

ERROR: The system cannot find the file specified.
Warning: running command 'schtasks /Delete /TN "getsurvey.R" /F' had status 1
Creating task schedule: schtasks /Create /TN "getsurvey.R" /TR "cmd /c C:/PROGRA~1/R/R-34~1.1/bin/Rscript.exe \"C:/Users/Administrator/Documents/R/FWP/getsurvey.R\"  >> \"C:/Users/Administrator/Documents/R/FWP/getsurvey.log\" 2>&1" /SC DAILY /ST 12:10 /SD "18/07/2017" 
Warning: running command 'schtasks /Create /TN "getsurvey.R" /TR "cmd /c C:/PROGRA~1/R/R-34~1.1/bin/Rscript.exe \"C:/Users/Administrator/Documents/R/FWP/getsurvey.R\"  >> \"C:/Users/Administrator/Documents/R/FWP/getsurvey.log\" 2>&1" /SC DAILY /ST 12:10 /SD "18/07/2017" ' had status 16389

If I run it as ONCE is runs fine.

Anything ideas?

Thanks!

**Comment by jwijffels** [comment]
the error messag indicates that the file C:/Users/Administrator/Documents/R/FWP/getsurvey.R does not exist on your computer

**Comment by avery021** [comment]
That is what I don't understand the file does exist. The file runs both once and hourly just not daily.

**Comment by jwijffels** [comment]
file.exists("C:/Users/Administrator/Documents/R/FWP/getsurvey.R")
make sure you don't give the same taskname twice.

**Comment by rvino** [comment]
I have the same issue, though it happens for any regularly scheduled task. If scheduled once it works fine, otherwise, I get the 16389 status error. I have checked the file path and it does exist:
`file.exists('R:/helloworld.R')
[1] TRUE`
If is schedule the task once it returns:
`Creating task schedule: schtasks /Create /TN "helloworld.R" /TR "cmd /c C:/Users/RORY_P~1/ONEDRI~1/MYDOCU~1/R/R-34~1.2/bin/Rscript.exe \"R://helloworld.R\"  >> \"R://helloworld.log\" 2>&1" /SC ONCE /ST 17:50 `

But if I schedule a regular refresh it returns:
`Creating task schedule: schtasks /Create /TN "helloworld.R" /TR "cmd /c C:/Users/RORY_P~1/ONEDRI~1/MYDOCU~1/R/R-34~1.2/bin/Rscript.exe \"R://helloworld.R\"  >> \"R://helloworld.log\" 2>&1" /SC MINUTE /ST 17:51 /SD "21/12/2017" 
Warning: running command 'schtasks /Create /TN "helloworld.R" /TR "cmd /c C:/Users/RORY_P~1/ONEDRI~1/MYDOCU~1/R/R-34~1.2/bin/Rscript.exe \"R://helloworld.R\"  >> \"R://helloworld.log\" 2>&1" /SC MINUTE /ST 17:51 /SD "21/12/2017" ' had status 16389`

I've tried moving the .R file into the C drive into a location without spaces as well and the same issue arises. Where the taskscheduleR library is, that file path does have spaces, but given that is where R is installed, I don't see an easy way around this?

**Comment by jwijffels** [comment]
Did you first deleted the task before creating a new one with the same name? `taskscheduler_delete("helloworld.R")` The README mentions:

`You can only have one task with the same name, make sure you use taskscheduler_delete the task if you are planning to create a new task with the same name.`

**Comment by rvino** [comment]
Yes, I have deleted the task first.

**Comment by jwijffels** [comment]
Then it must be the way you specify the startdate of your schedule. You need to use you locale.

**Comment by rvino** [comment]
Thank you for the help, it was the date issue.

**Comment by datasurfermtv** [comment]
Here's what i ended up doing: 

Additional arguments to RScript:
`startdate = format(Sys.Date()+1, "%m/%d/%Y")`

Change Date format of your locale:
 %m/%d/%Y
(note this is the way your machine's clock is formatted!)

**Comment by pinkumbrella14** [comment]
I am having the same issue, but I am using the RStudio Addin to schedule the tasks. How could I figure out what the format of my machine's clock is and where would I specify this in the Addin?
![taskscheduleR](https://user-images.githubusercontent.com/7611242/70498948-7d84f880-1ac4-11ea-88ab-e788b2e6bb80.PNG)

**Comment by jwijffels** [comment]
In 'Date format of your locale' in the rstudio add-in you can fill in something like %m/%d/%Y or %d/%m/%Y or whichever format you have set when you installed Windows

**Comment by liuyanguu** [comment]
Just figured out the same issue. When using RStudio Addins, you may need to change the Date format if NOT run once.
Indeed, this is not a problem if run only ONCE.

![image](https://user-images.githubusercontent.com/11966330/110339636-60581600-7ff6-11eb-88fc-6fbdfc9a65cb.png)

To best way to check is take a look at the task created, the "Next Run Time" should be the time you want the script to run for the first time. 
```
tasks <- taskscheduleR::taskscheduler_ls()
tasks[tasks$TaskName=="dashboard_update_ScheduleR.R",] # use your task name
```
![image](https://user-images.githubusercontent.com/11966330/110340104-e2e0d580-7ff6-11eb-95d4-767cc814ee67.png)
